spec/generator_spec.rb

```diff
@@ -14,32 +14,32 @@
   end
 
   context "without flags" do
-    let(:command) { "bundle exec rake install_all && bin/decidim #{test_app}" }
+    let(:command) { "bundle exec decidim #{test_app}" }
 
     it_behaves_like "a sane generator"
   end
 
   context "with --edge flag" do
-    let(:command) { "bin/decidim --edge #{test_app}" }
+    let(:command) { "bundle exec decidim --edge #{test_app}" }
```

Comment:
Similarly to the other comment in this file, the idea is not the use the development bundler context so it resembles better the real generator usage.